Committed test1 and Test2 class updates in -Tester1 brunch1

shamsevan opened the pull request:
HI .i am uploading my first brunch here. i have edited Test1 class and created test2 class. You will find the automation 
Testing regarding sign in functionality. Please review the codes. Thanks in advance.

Commit:
Tester1 - I have update test1 and Test2 class with some

Commit:
Tester1 - I have updated test1 and test2 class .and committing it in my
personal first brunch

shamsevan2 commented:
Hi x,
Your code looks good. but please check the condition steps again for making it easier.

Commit:
Hi Lead,
i have updated the Test1 class condition sections.

Commit:
Hi Lead,
I Have updated the test1 class and added more conditions in condition
section.You can review it again.

shamsevan commented:
Hi Lead Y, 
i have updated my codes , you can review it now. 
Thanks. Big regards .

shamsevan2 commented:
Hi Tester1, 
your code is perfect now . you can merge it now. 
Thanks, .regards,
QA lead y.
